03/app.js

```diff
@@ -1,3 +1,11 @@
 export default function randomNumber(min, max) {
+    if(typeof min !== 'number' || typeof max !== 'number') {
```

Comment:
Warto sprawdzać każdy z argumentów osobno, aby wiedzieć który jest niepoprawny. Teraz tego nie wiemy. Mimo, że napiszemy, że `randomNumber(1, 'max')` to nie wiemy, co powoduje błąd. Ponieważ pisząc `randomNumber('min', 1)` efekt będzie ten sam.